presto-benchmark/src/main/java/com/facebook/presto/benchmark/ArrayFunctionsBenchmark.java

```diff
@@ -0,0 +1,22 @@
+package com.facebook.presto.benchmark;
```

Comment:
Maybe we don't need this test anymore. Just remove it. The other benchmark is better.